modules/lang-painless/src/yamlRestTest/resources/rest-api-spec/test/painless/20_scriptfield.yml

```diff
@@ -88,6 +132,21 @@ setup:
 
     - match: { hits.hits.0.fields.bar.0: 8}
 
+---
+"Scripted Field with a null safe dereference (non-null, fields api)":
+  - do:
+      search:
+        rest_total_hits_as_int: true
+        body:
+          script_fields:
+            bar:
+              script:
+                source: "(field('foo').getValue('')?.length() ?: 0) + params.x;"
```

Comment:
why `getValue('')?.length()` here?

Reply:
Good catch. This one doesn't make sense, so I removed it.